### PR TITLE
[toolbars-v3] use context

### PR DESCRIPTION
**Pull request by noamtwix**

### CODE REVIEW

File: packages/toolbars-v2/web/src/toolbarItemConfig/draftToolbarItemConfig.ts [hunk 1]
```diff
@@ -262,6 +269,100 @@ export const draftStaticToolbarConfig: IToolbarItemConfig[] = [
...
+      visible: alwaysVisibleResolver,
+      selectedFontSize: getFontSizeInSelectionResolver,
+    },
+    commands: {
+      selectOption:
+        ({ editorCommands }) =>
+        value => {
+          if (!value) return;
+          const data = {
+            fontSize: value < 1 ? 1 : value > 900 ? 900 : value,
```
Comment (justame) [inline packages/toolbars-v2/web/src/toolbarItemConfig/draftToolbarItemConfig.ts:327]: 900? magic number?

Comment (noamtwix) [reply packages/toolbars-v2/web/src/toolbarItemConfig/draftToolbarItemConfig.ts:327]: this is the definition of the product

Comment (oded-soffrin) [reply packages/toolbars-v2/web/src/toolbarItemConfig/draftToolbarItemConfig.ts:327]: off-topic but:
```suggestion
            fontSize: Math.min(Math.max(1, value), 900),
```

Comment (justame) [reply packages/toolbars-v2/web/src/toolbarItemConfig/draftToolbarItemConfig.ts]: https://moderatemisbehaviour.github.io/clean-code-smells-and-heuristics/general/g25-replace-magic-numbers-with-named-constants.html

**Pull request merged**